src/SingleProjectOverview.js

```diff
@@ -1,35 +1,85 @@
 import React, {Component} from 'react';
-import {Header, Button, Image, Progress, Statistic, Grid} from "semantic-ui-react";
+import {
+    Header,
+    Button,
+    Image,
+    Progress,
+    Statistic,
+    Grid,
+    Dimmer,
+    Form,
+    Select,
+    Input,
+    Radio, TextArea, Checkbox, Container
+} from "semantic-ui-react";
 import "./SingleProjectOverview.css";
 
 const options = [
-    {key: 'f', text: 'Filter', value: 'filter'},
-    {key: 'p', text: 'Popularity', value: 'popularity'},
-    {key: 't', text: 'Targetbudget', value: 'targetbudget'},
-];
+    {key: 'm', text: 'Male', value: 'male'},
+    {key: 'f', text: 'Female', value: 'female'},
+]
 
 export class SingleProjectOverview extends Component {
-    constructor(props) {
```

Comment:
Constructor aanmaken met de props is wel nodig later (er kunnen meerdere projecten zijn die deze klasse gebruiken dus props moeten vanuit parent moeten worden doorgegeven).